reference/catalog.v3.yml

```diff
@@ -23904,13 +23904,13 @@ components:
           description: |
             A product can be added to an existing brand during a product /PUT or /POST.
         inventory_level:
-          maximum: 1000000000
+          maximum: 2147483647
```

Comment:
@bc-andreadao are these values in bytes? String lengths? Do we specify the unit anywhere?

Reply:
@slsriehl this is INT_MAX value https://learn.microsoft.com/en-us/cpp/cpp/integer-limits?view=msvc-170

Reply:
@slsriehl I believe the unit is implied because the type is integer. Or else we would have to move it to the description

Reply:
@bc-andreadao I verified the maximum value from:  https://learn.microsoft.com/en-us/cpp/cpp/integer-limits?view=msvc-170, this looks good